### ISSUE TITLE
Clean up Project Supports/Alpha testers

### ISSUE THREAD

**Issue by project64** [project64/web-site#3]
Project Supports was beta tester/people who donated before .. I was think of changing this to be beta testers and drop the alpha testing group.

**Comment by oddMLan** [comment]
Agreed, the alpha testing group seems moot right now.
BTW, if you decide to drop the alpha testing group do you plan to make the alpha subforum public (or merging it with the beta subforum)?

**Comment by project64** [comment]
yes it would have the same permissions as the beta group .. so like a merging
